lib/engine/game/base.rb

```diff
@@ -905,6 +905,10 @@ def sell_shares_and_change_price(bundle, allow_president_change: true, swap: nil
         log_share_price(corporation, price) if self.class::SELL_MOVEMENT != :none
       end
 
+      def destinations_toggle?
```

Comment:
i think you should be able to figure out if i game has destinations automatically by just doing

def destinations?
  @destinations ||= !@corporations.flat_map(&:destinations).empty?

Reply:
button is gone